src/test/java/core/basesyntax/FruitTransactionServiceTests.java

```diff
@@ -0,0 +1,110 @@
+package core.basesyntax;

```

Comment:
For the future, the structure for the tests should be the same as the structure of the project with all packages(sevice, model, strategy....)